life_server/Functions/Systems/fn_cleanup.sqf

```diff
@@ -4,19 +4,27 @@
     Author: Bryan "Tonic" Boardwine
 
     Description:
-    Server-side cleanup script on vehicles.
+    Server-side cleanup script on vehicles, dealers and fed reserve.
 */
 private _saveFuel = LIFE_SETTINGS(getNumber,"save_vehicle_fuel") isEqualTo 1;
 private _minUnitDistance = LIFE_SETTINGS(getNumber,"vehicles_despawn_max_distance");
 
-for "_i" from 0 to 1 step 0 do {
-    uiSleep (60 * 60);
+private _fnc_fedDealerUpdate = {
+    {
+        _x setVariable ["sellers",[],true];
```

Comment:
```suggestion
        private _dealer = missionNamespace getVariable [_x, nil];
        if (!isNil "_dealer") then {
            _dealer setVariable ["sellers", [], true];
        };
```

Reply:
if you do `missionNamespace getVariable [_x, nil];` you could also just go with `missionNamespace getVariable _x;`
default value makes only sense it its not nil

Reply:
potayto, potahto :)

Reply:
just unnecessary code in this case :D